plugins/modules/zos_operator.py

```diff
@@ -149,26 +154,28 @@ def parse_params(params):
 
 
 def run_operator_command(params):
+    module = AnsibleModule(argument_spec={}, check_invalid_arguments=False)
     command = params.get("cmd")
-    verbose = params.get("verbose")
-    debug = params.get("debug")
-    rc_message = []
-    rc_message = OperatorCmd.execute(command, "", verbose, debug)
-    rc = rc_message.get("rc")
-    message = rc_message.get("message")
+    verbose = "-v" if params.get("verbose") else ""
+    debug = "-d" if params.get("debug") else ""
+    rc, stdout, stderr = module.run_command(
+        "opercmd {0} {1} {2}".format(verbose, debug, quote(command)),
+        use_unsafe_shell=True,
```

Comment:
@blakeinate Is using `use_unsafe_shell` absolutely necessary? Not sure if it would be required for `opercmd`, just curious.

Reply:
Good question @asifmahmud, I made these changes a few weeks ago and will have to double check if there was a good reason using `use_unsafe_shell`. Will have to verify later today.

I do know the only piece of the command which is taken from the user is quoted, however, if we can get rid of `use_unsafe_shell` I am all for it 😄

Reply:
I was unable to reproduce any unexpected behavior after removing `use_unsafe_shell`. It has been removed and changes committed. Nice catch @asifmahmud !